File: src/main/java/com/uapp_llc/dto/task/TaskDto.java
package com.uapp_llc.dto.task;

import java.util.Date;

import lombok.Getter;
import lombok.Setter;

import com.uapp_llc.dto.column.ColumnDto;

@Getter
@Setter
public class TaskDto {

  private Long id;
  private Date createdAt;
  private String name;
  private String description;
  private ColumnDto column;

}

File: src/main/kotlin/com/uapp_llc/dto/task/TaskDto.kt
package com.uapp_llc.dto.task;

import java.util.Date
import com.uapp_llc.dto.column.ColumnDto

class TaskDto(

    var id: Long,
    var createdAt: Date,
    var name: String,
    var description: String,
    var index: Int,
    var column: ColumnDto

)
